### PR TITLE
feat(HMS-3924): add id to wizard's cancel button

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

File: src/Components/ProvisioningWizard/index.js [hunk 1]
```diff
@@ -78,6 +78,7 @@ const ProvisioningWizard = ({ onClose, hasAccess, image, ...props }) => {
         onNext={onNext}
         className={'provisioning'}
         footer={<CustomFooter />}
+        closeButtonAriaLabel='Close launch wizard'
```
Comment (amirfefer) [inline src/Components/ProvisioningWizard/index.js:81]: Besides the cancel button we also have the global `X` button for closing the wizard.
unfortunately there is no way to add an `id` prop to it, but we can add an aria label, or remove it completely. 
@ezr-ondrej. WDYT?

Comment (ezr-ondrej) [reply src/Components/ProvisioningWizard/index.js:81]: Lets keep it, cancel is good enough for now.
I'll communicate it to the UX team and decide, we can do follow-up for the `X` button :)

File: src/Components/ProvisioningWizard/CustomFooter.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ const CustomFooter = () => (
             )}
             {!activeStep.hideCancelButton && (
               <div className={styles.wizardFooterCancel}>
-                <Button variant={ButtonVariant.link} onClick={onClose}>
+                <Button id={`wizard-provisioning-cancel-${stepIdToString(activeStep.id)}`} variant={ButtonVariant.link} onClick={onClose}>
```
[the remark this answers is not in the document]
Comment (ezr-ondrej) [reply src/Components/ProvisioningWizard/CustomFooter.js:37]: IMO this is good :+1:

**Review by ezr-ondrej — APPROVED** [review]
Thank you @amirfefer ! :pray:

**Comment by ezr-ondrej** [comment]
I'll ignore the failures here :)

**Pull request merged**